[2024-05-28T03:46:04.087+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T03:46:04.118+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_files_batch.download_and_upload_task_0 scheduled__2024-05-28T03:44:00+00:00 [queued]>
[2024-05-28T03:46:04.128+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_files_batch.download_and_upload_task_0 scheduled__2024-05-28T03:44:00+00:00 [queued]>
[2024-05-28T03:46:04.129+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-28T03:46:04.165+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_task_0> on 2024-05-28 03:44:00+00:00
[2024-05-28T03:46:04.171+0000] {standard_task_runner.py:63} INFO - Started process 140 to run task
[2024-05-28T03:46:04.175+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'download_files_batch', 'download_and_upload_task_0', 'scheduled__2024-05-28T03:44:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/download-files_batch.py', '--cfg-path', '/tmp/tmpfzibdt8v']
[2024-05-28T03:46:04.180+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask download_and_upload_task_0
[2024-05-28T03:46:04.255+0000] {task_command.py:426} INFO - Running <TaskInstance: download_files_batch.download_and_upload_task_0 scheduled__2024-05-28T03:44:00+00:00 [running]> on host 7f87f2070cf0
[2024-05-28T03:46:04.389+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='download_files_batch' AIRFLOW_CTX_TASK_ID='download_and_upload_task_0' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T03:44:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-28T03:44:00+00:00'
[2024-05-28T03:46:04.390+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T03:46:04.406+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-05-28T03:46:47.233+0000] {logging_mixin.py:188} INFO - File downloaded successfully from https://covid.ourworldindata.org/data/owid-covid-data.csv!
[2024-05-28T03:46:48.744+0000] {logging_mixin.py:188} INFO - File our-world-in-data-covid19.csv uploaded successfully to MinIO!
[2024-05-28T03:46:48.745+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-28T03:46:48.745+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T03:46:48.754+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=download_files_batch, task_id=download_and_upload_task_0, run_id=scheduled__2024-05-28T03:44:00+00:00, execution_date=20240528T034400, start_date=20240528T034604, end_date=20240528T034648
[2024-05-28T03:46:48.804+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-28T03:46:48.817+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-28T03:46:48.818+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
